__tests__/topic.tests.js

```diff
@@ -105,3 +105,32 @@ describe("GET /api/articles/:articles_id", () => {
       });
   });
 });
+describe("GET /api/articles", () => {
+  test("Responds with an array of all articles object", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const articles = body.articles;
+        expect(Array.isArray(articles)).toBe(true);
+      });
+  });
+  test("Responds with an array of correct data types", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        //console.log(body, "<----- GET /api response");
+        const articles = body.articles;
+        articles.forEach((article) => {
```

Comment:
When making assertions inside of a forEach, it's important to check the length, to make sure it's not 0.

If the articles array was empty, then the tests wouldn't run. This would mean the test would pass as it wasn't given a change to fail